org.gnome.Games.LibretroPlugin.PicoDrive.metainfo.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Adrien Plazas -->
+<component type="addon">
+  <id>org.gnome.Games.LibretroPlugin.PicoDrive</id>
+  <extends>org.gnome.Games.desktop</extends>
```

Comment:
```suggestion
  <extends>org.gnome.Games</extends>
```
Should be the id of the app that it extends

Reply:
Unless Games uses that as an id in the metainfo

Reply:
Nope, no .desktop there.